roles/os_update_latest/tasks/main.yml

```diff
@@ -1,3 +1,10 @@
 ---
+- name: Make room for new kernel on /boot
+  command: /usr/bin/package-cleanup --oldkernels --count=2 -y
+
 - name: Update all packages
-  package: name=* state=latest
+  yum:
```

Comment:
Is the switch from `package` to `yum` intentional? We need to support Fedora installs where `dnf` is used and `package` module provides that abstraction.

Reply:
Nope, not intentional - I'll fix that.